lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1425,20 +1421,19 @@ def self.notify
           :description => <<~DESCRIPTION
             If `true`, disables agent middleware for Sinatra. This middleware is responsible for advanced feature support such as [cross application tracing](/docs/apm/transactions/cross-application-traces/cross-application-tracing), [page load timing](/docs/browser/new-relic-browser/getting-started/new-relic-browser), and [error collection](/docs/apm/applications-menu/events/view-apm-error-analytics).
 
-                <Callout variant="important">
-                  Cross application tracing is deprecated in favor of [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing). Distributed tracing is on by default for Ruby agent versions 8.0.0 and above. Middlewares are not required to support distributed tracing.
+            <Callout variant="important">
+            Cross application tracing is deprecated in favor of [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing). Distributed tracing is on by default for Ruby agent versions 8.0.0 and above. Middlewares are not required to support distributed tracing.
 
-                  To continue using cross application tracing, update the following options in your `newrelic.yml` configuration file:
+            To continue using cross application tracing, update the following options in your `newrelic.yml` configuration file:
 
-                  ```yaml
-                  # newrelic.yml
-
-                    cross_application_tracer:
-                      enabled: true
-                    distributed_tracing:
-                      enabled: false
-                  ```
-                </Callout>
+            ```yaml
+            # newrelic.yml
+            cross_application_tracer:
+              enabled: true
+            distributed_tracing:
+              enabled: false
+            ```
+            </Callout>
```

Comment:
This feels kind of random here. In the `newrelic.yml`, it makes it hard to tell what the config is about. I wonder if we can take it out entirely? There's a different deprecation warning that should print to the logs if someone is using CAT.

Reply:
I'll remove it!

Reply:
removed: https://github.com/newrelic/newrelic-ruby-agent/pull/3102/commits/5441e329923bfe07943c66e629d783cbcaac2c4f